front/app/modules/commercial/content_builder/admin/components/ContentBuilderToggle/ContentBuilderToggle.test.tsx

```diff
@@ -1,13 +1,30 @@
+import {
+  addContentBuilderLayout,
+  IContentBuilderLayoutObject,
+} from 'modules/commercial/content_builder/services/contentBuilder';
 import React from 'react';
 import { Multiloc } from 'typings';
 import { render, screen } from 'utils/testUtils/rtl';
 import ContentBuilderToggle from './';
 
+const mockContentBuilderLayoutData = {
```

Comment:
Let's also add a test for when this is an error. The way to do it is to turn this into a `let` and reassign it in your test:
```
mockContentBuilderLayoutData = new Error()```

Reply:
@IvaKop Should this be a separate test? Or added to the existing ones?

Reply:
Separate one - you are testing a separate test case.

Reply:
@IvaKop When I try to reassign mockContentBuilderLayoutData, I get the following error. I'm not quite sure what the correct way to initialize mockContentBuilderLayoutData would be? Should it be an empty json object first?
![Screenshot from 2022-04-01 11-17-41](https://user-images.githubusercontent.com/33987955/161234616-07086c8a-64bf-41fd-8e96-2f544dd10567.png)